### ISSUE TITLE
Focusing on a button that opens a modal and holding down Enter causes multiple backdrops

### ISSUE THREAD

**Issue by DannyJJK** [twbs/bootstrap#18208]
This is easiest to replicate on Firefox but it can be replicated on Chrome if you click the button then quickly hold down enter. Focusing on the button and holding down enter causes the modal dialog to be quickly opened and closed repeatedly as expected, however the background of the modal doesn't disappear correctly, and if you use the default transparent dark background, it turns into a black color because of how many additional backgrounds appear. Sometimes the modal becomes unclosable as well (clicking the X does nothing). I assume this is a setTimeout issue.

**Comment by cvrebert** [comment]
@DannyJJK Please post a JS Bin or JS Fiddle that demonstrates the problem.

**Comment by DannyJJK** [comment]
http://jsfiddle.net/5uvujt7j/

If you can't tab onto the first button, click the second button to focus onto it. On Chrome, if you click on the second button then quickly hold down enter, it should replicate the issue. If not, try closing the modal and try again (sometimes it doesn't work the first time). I have also included a third button that simulates the issue with javascript (toggle modal several times)

**Comment by twbs-lmvtfy** [comment]
Hi @DannyJJK!

You appear to have posted a live example (http://fiddle.jshell.net/5uvujt7j/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 29, column 3: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 30, column 1: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 31, column 1: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by DannyJJK** [comment]
I've updated the jsfiddle adding a type attribute to each button

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
